docs/workflow/ci.rst

```diff
@@ -0,0 +1,75 @@
+******************
+Continuous Integration
+******************
+
+In order to avoid any malfunctions after integrating new code, TARDIS is tested
```

Comment:
I would say what we do - not necessarily why we do it first. `We use a so-called Continuous Integration workflow with TARDIS  (link to page https://en.wikipedia.org/wiki/Continuous_integration). This means that each time a change is proposed (via pull-request) or a proposed change is merged into the main master branch a service will download the version and execute all the unit tests and integration tests offered with TARDIS.`